src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -256,9 +255,7 @@ private IOperation CreateInternal(BoundNode boundNode)
                     return CreateBoundIsPatternExpressionOperation((BoundIsPatternExpression)boundNode);
```

Comment:
Tagging @CyrusNajmabadi - I will file a separate issue to track this as this is not part of this PR.